Client/Frontend/Intro/OnboardingCardViewController.swift

```diff
@@ -141,6 +141,12 @@ class OnboardingCardViewController: UIViewController {
         updateLayout()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        viewModel.sendCardViewTelemetry()
```

Comment:
Hmm are we ok to send telemetry here as it might get called multiple times instead of did load? 🤔

Reply:
yes for this case because of the swipe option to go back and forward we want to log in viewWillAppear instead of viewDidLoad